lib/config/fileconf.go

```diff
@@ -989,9 +989,13 @@ func (s *Service) Disabled() bool {
 type Auth struct {
 	Service `yaml:",inline"`
 
-	// ProxyProtocol enables support for HAProxy proxy protocol version 1 when it is turned 'on'.
-	// Verify whether the service is in front of a trusted load balancer.
-	// The default value is 'on'.
+	// ProxyProtocol controls support for HAProxy PROXY protocol.
+	// Possible values:
+	// - 'on': one PROXY header is accepted and required per incoming connection.
+	// - 'off': no PROXY headers are allows, otherwise connection is rejected.
+	// If unspecified - one PROXY header is allowed, but not required. Connection is marked with source port set to 0.
```

Comment:
Please add a note that the proxy protocol not being specified is incorrect and IP pinning won't work.
It's also important to mention that you should configure it based on your L4 configs if there is a L4 at all.

Reply:
I've made same description here and in multiplexer, adding the details you've mentioned.